chain-tx-enclave-next/tdbe/enclave-app/src/sgx_module.rs

```diff
@@ -16,22 +16,21 @@ use thread_pool::ThreadPool;
 use webpki::DNSNameRef;
 
 use chain_core::tx::data::TxId;
+use enclave_macro::mock_key;
 use enclave_protocol::{
     codec::{StreamRead, StreamWrite},
     tdbe_protocol::{PersistenceCommand, TrustedTdbeRequest, TrustedTdbeResponse},
 };
+use enclave_utils::tls::{create_ra_context, create_tls_client_stream, create_tls_server_stream};
 use enclave_utils::SealedData;
 use ra_client::{EnclaveCertVerifier, EnclaveCertVerifierConfig, EnclaveInfo};
 use ra_enclave::{EnclaveRaConfig, EnclaveRaContext, DEFAULT_EXPIRATION_SECS};
 use tdbe_common::TdbeStartupConfig;
 
 const THREAD_POOL_SIZE: usize = 4;
 
-pub fn entry() -> std::io::Result<()> {
-    // Initialize logger
-    std::env::set_var("RUST_LOG", "debug");
-    env_logger::init();
-
+/// for reference -- to be changed with
+fn old_dummy_fetching_code() -> std::io::Result<()> {
```

Comment:
What do you mean by `old_dummy_fetching_code`?

Reply:
the previous non-working / incomplete code -- it'll be used / modified later with https://github.com/crypto-com/chain/issues/2182 https://github.com/crypto-com/chain/issues/2183 
I can put a feature-guard there instead

Reply:
Excluding/feature guarding isn't necessary here. If we pass `remote_rpc_address = None` in `TdbeConfig` (which is the value currently being passed), it won't execute the catch-up process code?

Reply:
ok, will look into that -- may need to modify the multi-node spawning, so that they don't collide on tdbe port listening

Reply:
I've put a bool flag in the initial options -- I guess it will need some other options, e.g. for "genesis" workflow ... one won't want to launch the listener before genesis.json is generated